api/views/healthCheck/healthCheck.py

```diff
@@ -0,0 +1,307 @@
+# Copyright The IETF Trust 2020, All Rights Reserved
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+__author__ = "Miroslav Kovac"
+__copyright__ = "Copyright The IETF Trust 2020, All Rights Reserved"
+__license__ = "Apache License, Version 2.0"
+__email__ = "miroslav.kovac@pantheon.tech"
+
+import json
+import MySQLdb
+import requests
+import time
+
+import utility.log as log
+
+from elasticsearch import Elasticsearch
+from flask import Blueprint, request, make_response, jsonify
+from api.globalConfig import yc_gc
+from utility.util import create_signature
+
+
+class HealthcheckBlueprint(Blueprint):
+
+    def __init__(self, name, import_name, static_folder=None, static_url_path=None, template_folder=None,
+                 url_prefix=None, subdomain=None, url_defaults=None, root_path=None):
+        self.LOGGER = log.get_logger('healthcheck', '/var/yang/logs/healthcheck.log')
+        super().__init__(name, import_name, static_folder, static_url_path, template_folder, url_prefix, subdomain,
+                         url_defaults, root_path)
+
+
+
+app = HealthcheckBlueprint('healthcheck', __name__)
+
+
```

Comment:
All the healthcheck endpoints should be work only if admin session exists or the password is provided